src/constant_texts.c

```diff
@@ -377,10 +377,9 @@ const char *ui_text_creation_failed_delete_wallet =
     "Wallet creation failed\nProceed to delete";
 
 const char *ui_text_successfull = "Successful";
-const char *ui_text_view_seed_messages[] = {
-    "Do not give the seed phrase to anyone",
-    "You will lose your coins if the seed phrase gets stolen",
-    "Press the center button to proceed"};
+const char *ui_text_view_seed_messages =
+    "Do not give the seed phrase to anyone. You will lose your coins if the "
+    "seed phrase gets stolen";
```

Comment:
Add: "Do you want to proceed?" in the message

Reply:
added